modular_skyrat/modules/cargo/code/packs.dm

```diff
@@ -125,6 +125,30 @@
 	crate_name = "unmarked weapons crate"
 	dangerous = TRUE
 	
+/datum/supply_pack/security/wildcat
+	name = "CFA Wildcat Supply Crate"
+	desc = "Local police running low on equipment? Neighbor particularly loud in the early morning? This should help out. (Does not include weapons permits.)"
+	cost = CARGO_CRATE_VALUE * 55 // Bit cheaper then the next up, because it's not to good.
+	contains = list(/obj/item/gun/ballistic/automatic/cfa_wildcat,
+					/obj/item/gun/ballistic/automatic/cfa_wildcat,
+					/obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat,
+					/obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat,
+					/obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat,
+					/obj/item/ammo_box/magazine/multi_sprite/cfa_wildcat)
+	crate_name = "CFA sponsored weapons crate"
+	dangerous = TRUE
+
+/datum/supply_pack/security/lasergunM // not to be confused with the normal laser gun crate
```

Comment:
It also doesn't follow naming conventions of snake_case.

Reply:
1, it's unseen and used nowhere ingame besides for the 'spawn cargo' command. 2. laserguns don't use snakecase anywhere else in the repo, so it'd be out of place to use it in this sole spot.

Reply:
> 1, it's unseen and used nowhere ingame besides for the 'spawn cargo' command.
Not a good reason for a poor name

> 2. laserguns don't use snakecase anywhere else in the repo, so it'd be out of place to use it in this sole spot.
I'd rather change the standard than keep it shit

Reply:
1. i overwrote the other lasergun crate instead
2. I'm not breaking the naming conventions for something non-modular because two people don't like it, you can feel free to PR upstream to change all instances of lasergun/lasgun/laser to laser_gun, but for consistency's sake I'm leaving it how it's setup too not add undue confusion with the already 4 different ways laserguns are set up codewise.